ctru-rs/src/services/cam.rs

```diff
@@ -0,0 +1,1020 @@
+use crate::services::gspgpu::FramebufferFormat;
+use bitflags::bitflags;
+use ctru_sys::Handle;
+use std::cell::RefCell;
+
+pub struct Cam {
```

Comment:
This should be `#[non_exhaustive]`, so that it can't be created by users manually.